aws/cloud-formation-template.yml

```diff
@@ -871,10 +871,11 @@ Resources:
                   --no-cache-dir --compile --ignore-installed \
                   pycurl
                 SLUGIFY_USES_TEXT_UNIDECODE=yes pip3 install \
-                  apache-airflow[celery,postgres,s3]==1.10.2 \
+                  apache-airflow[celery,postgres,s3,crypto]==1.10.2 \
                   celery[sqs] \
                   billiard==3.5.0.4 \
-                  tenacity==4.12.0
+                  tenacity==4.12.0 \
+                  cryptography
```

Comment:
Sorry for not being completely clear before, what I meant is that you don't need this line at all. IIRC adding `crypto` to `apache-airflow[...]` will install it already, in a version compatible.

Reply:
```suggestion
```